env-example

```diff
@@ -80,3 +80,8 @@ APPLE_API_BRIDGE_URL=
 # slack notifications
 SLACK_SNS_TOPIC=
 SLACK_CHANNEL_ID=
+
+# encrypt active record attributes
+ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY=0123456789abcdefghijklmnopqrstuv
+ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY=abcdefghijklmnopqrstuv0123456789
+ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT=abcdefghijk0123456789lmnopqrstuv
```

Comment:
DB encryption ftw!

Reply:
which is a good reminder I need to deploy values for these to the Infrastructure repo....